app/Exports/ApplicantsExport.php

```diff
@@ -81,7 +81,9 @@ public function map($application): array
                 return $studyLine->getNameWithYear();
             })->implode(', '),
             implode(",", $user->faculties->pluck('name')->toArray()),
-            implode(",", $user->workshops->pluck('name')->toArray()),
+            $application->applicationWorkshops->map(function ($workshop) {
```

Comment:
Maybe it would be a bit more efficient to create a new BelongsToMany relationship between applications and workshops (where the connection table is the table of ApplicationWorkshops) and querying them – that way, it's only one request.

Reply:
How about this?